### DISCUSSION TITLE
add pie chart at OrderDetailReport.cshtml using chart.js

### DISCUSSION THREAD

**Discussion by jiangws88** [serenity-is/Serenity#6949] [Help]
HI Guys,

I'm new for Serenity. 
I want to add pie chart at pdf report (export-pdf-button , using WKHTMLToPdf? ) as below:
1) put script at head:
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@3.4.1/dist/css/bootstrap.min.css" />
   <script src="~/lib/chartjs/chart.min.js"></script>
   <script>
       document.addEventListener('DOMContentLoaded', (event) => {
            var canvas = document.getElementById('pieChart');
           var ctx = canvas.getContext('2d');

           var pieData = {
               labels: ['Category 1', 'Category 2'],
               datasets: [{
                   data: [70, 30], 
                   backgroundColor: ['#FF5733', '#33FF57'] 
               }]
           };

           var myPieChart = new Chart(ctx, {
               type: 'pie',
               data: pieData,
               options: {
                   responsive: false, 
               }
           });

          });
   </script>
2) I can see pie at html format:

![image](https://github.com/serenity-is/Serenity/assets/140736331/a9382828-5815-4e18-861a-b97f03bf4423)

3) But can't see it at PDF file:
![image](https://github.com/serenity-is/Serenity/assets/140736331/2a4e092f-227d-4f7e-bd16-de65e604ad6f)

Please advise.

Thank you.

**Comment by jiangws88** [comment]
Solved.

**Comment by SuchitraPrab** [comment]
HI,

Can you please share the solution of export chart to pdf
